README.md

```diff
@@ -29,6 +29,90 @@ cd forms-admin
 ./bin/setup
 ```
 
+### Running the app
+
+You can either run the development task:
+
+```bash
+# Run the foreman dev server. This will also start the frontend dev task
+bin/dev
+```
+
+or run the rails server:
+
+```bash
+# Run a local Rails server
+bin/rails server
+
+# When running the server, you can use any of the frontend tasks, e.g.:
+npm run dev
+```
+
+## Development tools
+
+### Running the tests
+
+The app tests are written with [rspec-rails] and can be run with:
+
+```bash
+bundle exec rspec
+```
+
+There are also unit tests for JavaScript code (look for files named `*.test.js`), written with [Jest]. These can be run with:
+
+```bash
+npm run test
+```
+
+[rspec-rails]: https://github.com/rspec/rspec-rails
+[Jest]: https://jest.io
+
+### Linting
+
+We use [RuboCop GOV.UK] for linting code, to autocorrect issues run:
+
+```bash
+bundle exec rubocop -A
+```
+
+We also use the [i18n-tasks] tool to keep our locales files in a consistent order. When the tests run they will check if the locale files are normalised, and fail if they are not. To fix the locale files automatically you can run:
+
+```bash
+bundle exec i18n-tasks normalize
+```
+
+On GitHub pull requests we also check our dependencies for security issues using [bundler-audit], you can run this locally with:
+
+```bash
+bundle audit
+```
+
+[RuboCop GOV.UK]: https://github.com/alphagov/rubocop-govuk
+[i18n-tasks]: https://github.com/glebm/i18n-tasks
+[bundle-audit]: https://github.com/rubysec/bundler-audit
+
+### Running tasks with rake
+
+We have a [Rakefile](./Rakefile) that is setup to follow the [GOV.UK conventions for Rails applications].
```

Comment:
We have a [Rakefile](./Rakefile) that is set up to follow the [GOV.UK conventions for Rails applications].